whipper/common/encode.py

```diff
@@ -19,7 +19,8 @@
 # along with whipper.  If not, see <http://www.gnu.org/licenses/>.
 
 
-from mutagen.flac import FLAC
+from mutagen.flac import FLAC, Picture
+from mutagen.id3 import PictureType
```

Comment:
Why do we need `PictureType` from `….id3`? Is this also needed for FLAC files?

Reply:
@Freso , `PictureType` is used [here](https://github.com/whipper-team/whipper/pull/436/files#diff-6d77faf66a5bcf0caa2a26f3b6f0d0f0R140). Just like the code written in [mutagen.flac.Picture's doc](https://mutagen.readthedocs.io/en/latest/api/flac.html#mutagen.flac.Picture) :
```
pic.type = id3.PictureType.COVER_FRONT
pic.mime = u"image/jpeg"
pic.width = 500
pic.height = 500
pic.depth = 16 # color depth
```